src/rest/VolumeEndpoint.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+
+/**
+ * Endpoint for GETing volume metrics
+ */
+module.exports = (volumeLogger) => {
```

Comment:
Now there's no authentication at all for this endpoint, but then again, `nginx` only routes regex `/api/v1/streams/.*/data(/.*)?` to data-api, so in the current setup it would not be publicly accessible in any case.

If we want to avoid accidentally exposing it in the future, we could add some kind of check to it. For example, it would be simple to only allow the endpoint to be accessed from `127.0.0.1`, but I'm not sure if we know where our metrics agent collecting this data will run?

Reply:
Actually checking ip address didn't seem to work very well because of proxying.

Reply:
I think we need some authentication in the future but for now it's ok. Also if we leak statistics it's not compromising.